docs/guides/dashboard/README.md

```diff
@@ -53,6 +53,4 @@ app: # A parent namespce for this Service and related Services App name (e.g. mo
 
 ```
 
-You must deploy your Service to have it show in the Dashboard with the following configuration in your YAML file. Run `serverless deploy` to do this.
-
-By default, Deployment history and Observability will be set up automatically. To enable Observability, Serverless Framework will automatically create an AWS IAM Role within the AWS account you deployed to, giving Serverless Inc permission to AWS Cloudwatch and more. [You can transparently see the permissions this role requires in Github](https://github.com/serverless/console/blob/main/instrumentation/aws/iam-role-cfn-template.yaml).
+You must deploy your Service with the above configuration in your YAML file to have it show in the Dashboard. Run `serverless deploy` to do this. The Serverless Framework will automatically create an AWS IAM Role within the AWS account you deployed to, giving Serverless Inc permission to AWS Cloudwatch and more. [You can transparently see the permissions this role requires in Github](https://github.com/serverless/console/blob/main/instrumentation/aws/iam-role-cfn-template.yaml). By default, Deployment history will be set up automatically.
```

Comment:
This doesn't make sense anymore. It was clear in the previous version about observability being set up by default. Now, it randomly talks about creating IAM Roles in their AWS account.

**Let's change to this:**

You must deploy your Service with the above configuration in your YAML file to have it show in the Dashboard. Run `serverless deploy` to do this. 

If you want to enable Observability (metrics, traces, logs) for your AWS Lambda functions within Serverless Framework Dashboard, you will have to add the following in the `stages` block of your `serverless.yml`:

```yaml
stages:
  prod:
    observability: true # turn on observability in the "prod" Stage.
  dev:
    observability: true # turn on observability in the "dev" Stage.
  default:
    observability: false # turn off observability for all other Stages.
```

To set up Observability, Serverless Framework will automatically create an AWS IAM Role within the AWS account you deployed to, giving Serverless Inc permission to AWS Cloudwatch and more. [You can transparently see the permissions this role requires in Github](https://github.com/serverless/console/blob/main/instrumentation/aws/iam-role-cfn-template.yaml).